src/qt/window.cpp

```diff
@@ -1680,3 +1716,31 @@ QPainter *wxWindowQt::QtGetPainter()
 {
     return m_qtPainter.get();
 }
+
+bool wxWindowQt::EnableTouchEvents(int eventsMask)
+{
+    if(GetHandle()){
```

Comment:
This probably should be
```suggestion
    wxCHECK_MSG( GetHandle(), false, "can't be called before creating the window" );
```
instead of silently doing nothing in this case.

Reply:
Agreed. Will fix.